sim/rogue/rupture.go

```diff
@@ -48,6 +48,12 @@ func (rogue *Rogue) registerRupture() {
 			Aura: core.Aura{
 				Label: "Rupture",
 				Tag:   RogueBleedTag,
+				OnGain: func(aura *core.Aura, sim *core.Simulation) {
+					rogue.BleedsActive++
+				},
+				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+					rogue.BleedsActive--
+				},
```

Comment:
And the `OnExpire`s would look like
```suggestion
				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
					rogue.BleedsActive[aura.Unit.TargetIndex]--
				},
```